### PR TITLE
DX-2564

### PR DESCRIPTION
**Pull request by DX-Bandwidth**
Update API specs from upstream api-specs repository. Opened By: @brianluisgomez

### CODE REVIEW

File: site/specs/global.yml [hunk 1]
```diff
@@ -905,8 +945,8 @@ tags:
       Apply Configuration method.
   - name: Connectivity
     description: >-
-      Retrieve details about Bandwidth's data centres and network interconnect
-      for various services such as call termination, speech analytics, etc.
+      Retrieve details about Voxbone's data centres and network interconnect for
```
Comment (bpateldx) [inline site/specs/global.yml:948]: There is plenty of reference to voxbone in the updated Global API spec. Is it after review and agreed upon or do we've separate ticket to review it?

Comment (ajrice6713) [reply site/specs/global.yml:948]: Yeah thats all addressed in [this](https://github.com/Bandwidth/api-specs/pull/127) PR

Additionally - any of the spec file changes are automatically pulled from [api-specs](https://github.com/Bandwidth/api-specs) - the source of truth for the PRs are in that repository 

@brianluisgomez didnt edit that `global.yml` file in his PR - it looks like its just getting pulled from main of api-specs 

running `git merge main` on this branch of api-docs should remove that `global` file change

**Comment by DX-Bandwidth** [comment]
Corresponding Pull Request on [api-specs](https://github.com/Bandwidth/api-specs/pull/172) was merged.

**Review by matthewkmartin — APPROVED** [review]

**Review by GmVhr — COMMENTED** [review]
JNE Express.

**Comment by GmVhr** [comment]
Saya suka

**Comment by GmVhr** [comment]
Saya suka

**Pull request merged**